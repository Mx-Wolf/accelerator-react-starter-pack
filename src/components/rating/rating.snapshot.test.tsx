import renderer from 'react-test-renderer';
import Rating from './rating';

const fakeRating = 2;

describe('Component: Rating', () => {
  it('should render correctly', () => {
    const component = renderer.create(<Rating guitarRating={fakeRating} />);
    expect(component).toMatchSnapshot();
  });
});
