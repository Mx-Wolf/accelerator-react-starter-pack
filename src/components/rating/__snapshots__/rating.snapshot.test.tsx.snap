// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Rating should render correctly 1`] = `
Array [
  <svg
    aria-hidden="true"
    height="11"
    width="12"
  >
    <use
      xlinkHref="#icon-full-star"
    />
  </svg>,
  <svg
    aria-hidden="true"
    height="11"
    width="12"
  >
    <use
      xlinkHref="#icon-full-star"
    />
  </svg>,
  <svg
    aria-hidden="true"
    height="11"
    width="12"
  >
    <use
      xlinkHref="#icon-star"
    />
  </svg>,
  <svg
    aria-hidden="true"
    height="11"
    width="12"
  >
    <use
      xlinkHref="#icon-star"
    />
  </svg>,
  <svg
    aria-hidden="true"
    height="11"
    width="12"
  >
    <use
      xlinkHref="#icon-star"
    />
  </svg>,
]
`;
